src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Packages/Analyzers/VbClassLibraryMyDotAnalyzer.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.DotNet.UpgradeAssistant.Dependencies;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.DotNet.UpgradeAssistant.Steps.Packages.Analyzers
+{
+    /// <summary>
+    /// This analyzer will modify the vbproj of class libraries to resolve compilation errors:
+    /// 1. Adding System.Configuration.ConfigurationManager resolves the error from Settings.Designer.vb
+    /// 2. Adding <VBRuntime>Embed</VBRuntime> resolves the error from the vb compiler
+    ///     Per https://github.com/dotnet/runtime/issues/30478#issuecomment-521270193.
```

Comment:
This seems to indicate it should be fixed and not necessary. Is this the right long term fix?

Reply:
Will follow up to see if there is another fix recommended. For now, this fix addresses the error reported.